src/components/PetsSelector.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { getPet, getPets } from "../service/petsAPI";
+import Select from "react-select";
+import { SelectorWrapper } from "../styled/SelectorWrapper.styled";
+
+export default function PetsSelector() {
+  const [page, setPage] = useState(1);
+  const [loading, setLoading] = useState(false);
+  const [selectorPets, setSelectorPets] = useState([]);
+
+  const [petID, setPetID] = useState(null);
+  const [petName, setPetName] = useState(null);
+  const [petStatus, setPetStatus] = useState(null);
+
+  useEffect(() => {
+    if (!petID) return;
+    const getPetInfo = async () => {
+      const petInfo = await getPet(petID);
+      setPetName(petInfo.name ? petInfo.name : "No name");
```

Comment:
Whenever the first part of a ternary is the same as the condition you can just use `or`, this is equivalent to `petInfo.name || "No name"`